.github/workflows/cucumber-test.yml

```diff
@@ -2,15 +2,15 @@ name: Cucumber Tests
 
 on:
   pull_request:
-    branches: [master]
+    branches: [main]
   push:
-    branches: [master]
+    branches: [main]
```

Comment:
Actually, here's a question.  Do we need to have the tests run when pushing to main?  Or only when someone creates a PR?  Presumbly when main is updated, it can only come from a PR, the PR will have passed all tests.  There may not need to be a need to run the tests again.

Reply:
Good thought, it does make sense. Is it possible that we might come across a situation where the PR is not up to date and be missing the changes when different PR's are merged in simultaneously?

Reply:
Once we have all tests and CICD working, we can force PRs to be up-to-date before merging.

Reply:
Yes, sounds good.